packages/rocketchat-lib/server/models/Messages.js

```diff
@@ -318,10 +320,34 @@ RocketChat.models.Messages = new class extends RocketChat.models._Base {
 		return this.insert(record);
 	}
 
+	findJoinRequestsByMsgTypeAndRoom(msgType, rid) {
```

Comment:
@vickyokrm still not nice: if there's a `msgType` in the  signature, `JoinRequest` it must not be part of the name. Since it's a method of the message-model, it should be `findByTypeAndRoom`

Reply:
@vickyokrm this is still pending, right?